sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -499,9 +505,10 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
         const messages = result.body.messages as { message: Buffer }[];
         for (const msg of messages) {
           const decodedMessage = RheaMessageUtil.decode(msg.message);
-          const message = fromRheaMessage(decodedMessage as any);
-
-          message.body = defaultDataTransformer.decode(message.body);
+          const message = fromRheaMessage(
+            decodedMessage as any,
+            options?.skipParsingBodyAsJson ?? false
```

Comment:
For management client, should I always pass `false` here and remove this option from the internal `SendManagementRequestOptions`

Reply:
but why?

Reply:
I thought that message format used by management client would remain the same thus wouldn't need any customization.

Reply:
nvm I probably was thinking about admin client which doesn't use amqp anyway.